pages/community/election.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: US-RSE Steering Committee Elections
+subtitle:
+bigimg:
+permalink: /election/
+---
+
+The 2021 US-RSE Steering Committee elections are happening this December.
+
+## Timeline
+
+The timeline for this year's elections will be posted soon.
+
+
+## Positions and Eligibility
+Five Steering Committee seats are open in this election, each for a 2-year term (January 1, 2022 through December 31, 2023). The following steering committee members have terms that end this year: Jeff Carver, Ian Cosden, Charles Ferenbaugh, Sandra Gesing, and Christina Maimone. All US-RSE members in good standing as of the day nominations open are eligible for nomination.  
+
+A membership verification email will be sent to all members in October. The message will be sent directly from leadership@us-rse.org, not through the Google group used for most email communications.   Individuals who do not receive the membership verification email may join the organization by filling out the membership form at [https://us-rse.org/join/](https://us-rse.org/join/).  For questions or help with membership matters, please contact Election Chairs Julia Damerow and Lance Parsons on Slack or [via email](mailto:jdamerow@asu.edu,lparsons@princeton.edu).
+
+The US-RSE Steering Committee sets its own meeting frequency and timing.  Currently, the Steering Committee is meeting every two weeks.  The Steering Committee meeting frequency and schedule for 2022 will be determined once new members are elected.  
+
+The responsibilities of Steering Committee members are detailed in the [US-RSE Membership and Governance](https://github.com/USRSE/documents/blob/master/governance.md) document. They include:
+
+* Setting the overall policy and direction of the US-RSE Association
+* Being responsible for day-to-day activities, operations, and development of the US-RSE Association
+
+Additionally, Steering Committee members are expected to:
+
+* Attend and participate in Steering Committee meetings
+* Follow and participate in Steering Committee discussions on Slack
+* Contribute to US-RSE operations through activities such as
+  * Updating the website
+  * Writing newsletters and communications
+  * Organizing and leading community calls and events
+  * Engaging with community members on Slack
+* Help further the community by
+  * Participating on international RSE committees
+  * Organizing BOFs, workshops, meetups, and other events at relevant conferences
+  * Leading US-RSE working groups
+  * Writing blog posts
+  * Bringing new ideas to the organization
+
+## Nominations and Candidate Information
+
+Individuals may nominate themself or another member.  In cases when a nomination is not a self-nomination, the Election Chair will contact the nominated individual to confirm their interest in participating.
+
+On the nomination form, the following information is required:
```

Comment:
Does the form exist? I think it should be linked and available when this goes live. That is, nominations are open once this goes live along with a clear end date for nominations.